com.toolstack.Folio.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id" : "com.toolstack.Folio",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.toolstack.Folio",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=host"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name" : "blueprint-compiler",
+            "buildsystem" : "meson",
+            "sources" : [{
+                "type" : "git",
+                "url" : "https://gitlab.gnome.org/jwestman/blueprint-compiler.git",
+                "branch" : "v0.10.0"
+            }]
+        },
+        {
+            "name" : "Folio",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [{
+                "type" : "git",
+                "url" : "https://github.com/toolstack/Folio",
+                "branch" : "main"
```

Comment:
never use git branch. You must use a tag or/and commit

Reply:
Doh, sorry, grabbed the wrong version, fixed.